### ISSUE TITLE
Não está resolvendo o captcha

### ISSUE THREAD

**Issue by joaomakonha** [DanielHe4rt/bbbot#37]
Aparentemente não estou conseguindo computar nenhum voto, é como se errasse todos os captchas.
![image](https://user-images.githubusercontent.com/62957459/78075098-493a8e00-737a-11ea-91ee-77c705282023.png)

**Comment by lucasdidur** [comment]
Mesma coisa comigo

**Comment by GiovanniCardoso9** [comment]
quando ele fala em executar run-server-login.bat nao é no terminal ? pra mim sempre da erro

**Comment by GiovanniCardoso9** [comment]
parece que miou

**Comment by allanzh** [comment]
A globo trocou as imagens, galera. Desde a hora da live de ontem já tava assim! o bot fica errando várias pra as vezes acertar uma.

**Comment by Mateuus** [comment]
Tem vezes que ele acerta 3 vezes, parece que a globo bloquear em tempo em tempo o voto

**Comment by allanzh** [comment]
Sim, estão bloqueando a conta também, o ideal é criar um mecanismo de revezamento de várias contas, alguém comentou que tava trabalhando nisso em um outro issue

**Comment by Mateuus** [comment]
e também o ip mesmo trocando de conta eles bloquea

**Comment by UriellViana** [comment]
<img width="381" alt="Screen Shot 2020-03-31 at 7 24 05 PM" src="https://user-images.githubusercontent.com/39530115/78081233-6d03d100-7386-11ea-84ab-66dedb40aabf.png">
A Globo lançou a braba e meteu um loading fake de 5s segundos na votação. Além disso o ratelimit (limite de requisições) da API de votos tá mto mais severo (e eu acho que por IP agora).

**Comment by RCMelo13** [comment]
Com essa alteração voltou a funcionar?
